keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperExtension.kt

```diff
@@ -108,4 +115,21 @@ public interface VariantFilter {
    * Returns the unique variant name.
    */
   public val name: String
-}
\ No newline at end of file
+}
+
+public abstract class TraceReferences @Inject constructor(objects: ObjectFactory) {
+  /**
+   * Controls whether or not to use the new experimental TraceReferences entry-point.
+   * Default is false but it's automatically enabled if the traceReferences block was used.
+   */
+  internal val enabled: Property<Boolean> = objects.property<Boolean>().convention(false)
+
+  /**
+   * Optional arguments during the trace-references invocation,
+   * only considered if [isTraceReferencesEnabled] is true.
+   *
+   * Default value: `listOf("--map-diagnostics:MissingDefinitionsDiagnostic", "error", "info")`
```

Comment:
Is there a link we could add here of where to find these dianostics?

Reply:
It's printed in the task stdout, so it's accessible in Gradle logs.
It's coming from the discussion I had here: https://issuetracker.google.com/issues/173435379

I'm not a big fan of this current setup but I kept it as default because having the plugin working out-of-the-box seems important for a good user experience.

Reply:
Let's link that issue then at least

Reply:
Done in 6647c8c